source/includes/_configuration-options.md

```diff
@@ -32,11 +32,29 @@ const configurationOptions = {
     hideSimilarProducts: false,
     useCustomCss: true
   },
-  useDeepLinking: false
+  useDeepLinking: false,
+  customLabels: {
+    buy: 'custom button label'
```

Comment:
maybe instead `custom button label`, let's use some example from another client like "Add to wardrobe"?

Reply:
I agree, it would be a much better and practical example.